api/src/main/resources/workbench.yaml

```diff
@@ -425,6 +425,41 @@ paths:
       summary: remove a user from an auth domain if you have manage groups permission
       operationId: removeUserFromAuthDomain
 
+  /v1/user/{term}:
+    get:
+      tags:
+        - user
+      description: Returns user information matching search terms
+      operationId: user
+      responses:
+        200:
+          description: The user information matching supplied search terms
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/UserResponse"
+      parameters:
```

Comment:
We have an existing paginated API defined here: https://github.com/all-of-us/workbench/blob/master/api/src/main/resources/client_api.yaml#L127 which follows the Google Cloud API style guide for pagination: https://cloud.google.com/apis/design/design_patterns#list_pagination

My preference would be to follow that pattern here - happy to talk at length about the merits of that approach. I'm also OK using an offset as the implementation for now to keep similarity with what you've already done (though I do think that approach is not ideal).

Reply:
Thank you - will look at that.